askama_shared/src/error.rs

```diff
@@ -81,6 +86,70 @@ impl From<::serde_yaml::Error> for Error {
     }
 }
 
+#[doc(hidden)]
```

Comment:
All this stuff seems really complex. Could we only add a `Custom(Box<dyn Error + Send + Sync + 'static>)` variant to `Error` and be done with it?

Reply:
Of course. :)